0-common/scala-core/project/IgluCoreBuild.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2016 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+import sbt._
+import Keys._
+
+object IgluCoreBuild extends Build {
+
+  import Dependencies._
+  import BuildSettings._
+
+  // Configure prompt to show current project
+  override lazy val settings = super.settings :+ {
+    shellPrompt := { s => Project.extract(s).currentProject.id + " > " }
+  }
+
+  // Define our project, with basic project information and library dependencies
+  lazy val project = Project("iglu-core", file("."))
+    .settings(buildSettings: _*)
+    .settings(
+      libraryDependencies ++= Seq(
+        // Scala (test only)
+        Libraries.json4s,
+        Libraries.specs2
+      )
+    )
+}
```

Comment:
I get this error when I try to `sbt compile`:

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.scalaz.stream#scalaz-stream_2.10;0.7a: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		org.scalaz.stream:scalaz-stream_2.10:0.7a
[warn] 		  +- org.specs2:specs2-common_2.10:3.3.1
[warn] 		  +- org.specs2:specs2-matcher_2.10:3.3.1
[warn] 		  +- org.specs2:specs2-core_2.10:3.3.1 (/vagrant/0-common/scala-core/project/IgluCoreBuild.scala#L30)
[warn] 		  +- com.snowplowanalytics:iglu-core:0.1.0-M1
```

This happens both in my host machine and in the Iglu VM I had made earlier. I haven't tried it inside a VM built using scala-core/Vagrantfile.